site/gatsby-site/server/remote.ts

```diff
@@ -0,0 +1,76 @@
+import { buildHTTPExecutor, HeadersConfig } from '@graphql-tools/executor-http'
+import { makeExecutableSchema } from '@graphql-tools/schema';
+import { FilterObjectFields, FilterTypes, wrapSchema } from '@graphql-tools/wrap';
+import remoteTypeDefs from './remoteTypeDefs';
+import config from './config';
+import { ExecutionRequest } from '@graphql-tools/utils';
+
+const userExecutor = buildHTTPExecutor({
+    endpoint: `https://realm.mongodb.com/api/client/v2.0/app/${config.REALM_APP_ID}/graphql`,
+    headers(executorRequest?: ExecutionRequest<any, any, any, Record<string, any>, any> | undefined): HeadersConfig {
+        if (
+            !executorRequest?.context.req.headers.authorization && executorRequest?.info?.operation.operation == 'query'
+        ) {
+            return {
+                apiKey: config.REALM_GRAPHQL_API_KEY,
+                authorization: '',
+            };
+        }
+
+        return {
+            apiKey: '',
+            authorization: executorRequest?.context.req.headers.authorization!,
+        };
+    },
+});
+
+
+const ignoreTypes = [
```

Comment:
As we gradually migrate to the API, we have to filter out fields and types coming from Atlas graphql.